source/Octopus.Manager.Tentacle.Tests/ConfiguringTentacleServiceFixture.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using FluentAssertions;
+using NSubstitute;
+using NUnit.Framework;
+using Octopus.Manager.Tentacle.Controls;
+using Octopus.Manager.Tentacle.DeleteWizard;
+using Octopus.Manager.Tentacle.Proxy;
+using Octopus.Manager.Tentacle.Shell;
+using Octopus.Manager.Tentacle.TentacleConfiguration.SetupWizard;
+using Octopus.Manager.Tentacle.TentacleConfiguration.TentacleManager;
+using Octopus.Manager.Tentacle.Tests.Utils;
+using Octopus.Manager.Tentacle.Util;
+using Octopus.Tentacle.Configuration;
+using Octopus.Tentacle.Configuration.Instances;
+using Octopus.Tentacle.Util;
+
+namespace Octopus.Manager.Tentacle.Tests.ModelFixtures
+{
+    [TestFixture]
+    public class ConfiguringTentacleServiceFixture
+    {
+        static readonly string CommandLinePath = CommandLine.PathToTentacleExe();
+        
+        [Test]
+        public void WhenStartingATentacle_ScriptShouldBeGeneratedCorrectly()
+        {
+            // Arrange
+            var model = CreateTestTentacleManagerViewModel();
+            
+            // Act
+            var script = model.ServiceWatcher.GetStartCommands().ToCommandLineString();
+
+            // Assert
+            var pathToTentacleExe = $"\"{CommandLinePath}\"";
+            var expectedOutput = $"{pathToTentacleExe} service --instance \"{model.InstanceName}\" --start";
+            script.Should().Be(expectedOutput);
+        }
+        
+        [Test]
+        public void WhenStoppingATentacle_ScriptShouldBeGeneratedCorrectly()
+        {
+            // Arrange
+            var model = CreateTestTentacleManagerViewModel();
+            
+            // Act
+            var script = model.ServiceWatcher.GetStopCommands().ToCommandLineString();
+
+            // Assert
+            var pathToTentacleExe = $"\"{CommandLinePath}\"";
+            var expectedOutput = $"{pathToTentacleExe} service --instance \"{model.InstanceName}\" --stop";
+            script.Should().Be(expectedOutput);
+        }
+
+        [Test]
+        public void WhenReinstallingATentacle_ScriptShouldBeGeneratedCorrectly()
+        {
+            // Arrange
+            var model = CreateTestTentacleManagerViewModel();
+            
+            // Act
+            var script = model.ServiceWatcher.GetRepairCommands().ToCommandLineString();
+
+            // Assert
+            var pathToTentacleExe = $"\"{CommandLinePath}\"";
+            var expectedOutput = $"{pathToTentacleExe} service --instance \"{model.InstanceName}\" --stop" +
```

Comment:
The multiple string concatenations feels a bit messy. Can we update to use a `StringBuilder` instead?